Adds comment lines which can be used as anchor points by
individual context patches applied to the file later.

Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -10,6 +10,7 @@
 # possible, but leave them commented.  Uncommented options change a
 # default value.
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_1
 #Port 22
 #AddressFamily any
 #ListenAddress 0.0.0.0
@@ -18,8 +19,10 @@
 # The default requires explicit activation of protocol 1
 #Protocol 2
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_2
 # HostKey for protocol version 1
 #HostKey /etc/ssh/ssh_host_key
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_3
 # HostKeys for protocol version 2
 #HostKey /etc/ssh/ssh_host_rsa_key
 #HostKey /etc/ssh/ssh_host_dsa_key
@@ -42,6 +45,7 @@
 #MaxAuthTries 6
 #MaxSessions 10
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_4
 #RSAAuthentication yes
 #PubkeyAuthentication yes
 #AuthorizedKeysFile	.ssh/authorized_keys
@@ -56,10 +60,12 @@
 # Don't read the user's ~/.rhosts and ~/.shosts files
 #IgnoreRhosts yes
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_5
 # To disable tunneled clear text passwords, change to no here!
 #PasswordAuthentication yes
 #PermitEmptyPasswords no
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_6
 # Change to no to disable s/key passwords
 #ChallengeResponseAuthentication yes
 
@@ -73,6 +79,7 @@
 #GSSAPIAuthentication no
 #GSSAPICleanupCredentials yes
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_7
 # Set this to 'yes' to enable PAM authentication, account processing, 
 # and session processing. If this is enabled, PAM authentication will 
 # be allowed through the ChallengeResponseAuthentication and
@@ -84,9 +91,12 @@
 # and ChallengeResponseAuthentication to 'no'.
 #UsePAM no
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_8
 #AllowAgentForwarding yes
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_9
 #AllowTcpForwarding yes
 #GatewayPorts no
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_10
 #X11Forwarding no
 #X11DisplayOffset 10
 #X11UseLocalhost yes
@@ -94,17 +104,22 @@
 #PrintLastLog yes
 #TCPKeepAlive yes
 #UseLogin no
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_11
 #UsePrivilegeSeparation yes
 #PermitUserEnvironment no
 #Compression delayed
 #ClientAliveInterval 0
 #ClientAliveCountMax 3
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_12
 #UseDNS yes
 #PidFile /var/run/sshd.pid
 #MaxStartups 10
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_13
 #PermitTunnel no
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_14
 #ChrootDirectory none
 
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_15
 # no default banner path
 #Banner none
 
@@ -130,3 +145,10 @@ Subsystem	sftp	/usr/lib64/misc/sftp-serv
 #	X11Forwarding no
 #	AllowTcpForwarding no
 #	ForceCommand cvs server
+
+# anchor_1i2w6k7g8dtsxb62ev3eliju4_16
+# Non-default options that have been added to the config file
+# ciphers: anchor_1i2w6k7g8dtsxb62ev3eliju4_17
+# macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_18
+# kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_19
+# EOF: anchor_1i2w6k7g8dtsxb62ev3eliju4_20
Index: ssh/ssh_config
===================================================================
--- ssh.orig/ssh_config
+++ ssh/ssh_config
@@ -35,13 +35,19 @@
 #   IdentityFile ~/.ssh/id_rsa
 #   IdentityFile ~/.ssh/id_dsa
 #   Port 22
+# protocol: anchor_1i2w6k7g8dtsxb62ev3eliju4_1
 #   Protocol 2,1
+# ciphers: anchor_1i2w6k7g8dtsxb62ev3eliju4_2
 #   Cipher 3des
 #   Ciphers aes128-ctr,aes192-ctr,aes256-ctr,arcfour256,arcfour128,aes128-cbc,3des-cbc
+# hkalgos: anchor_1i2w6k7g8dtsxb62ev3eliju4_3
+# macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_4
 #   MACs hmac-md5,hmac-sha1,umac-64@openssh.com,hmac-ripemd160
+# kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_5
 #   EscapeChar ~
 #   Tunnel no
 #   TunnelDevice any:any
 #   PermitLocalCommand no
 #   VisualHostKey no
 #   ProxyCommand ssh -q -W %h:%p gateway.example.com
+# EOF: anchor_1i2w6k7g8dtsxb62ev3eliju4_6
